curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/quoting-strings-with-single-quotes.md

```diff
@@ -43,7 +43,7 @@ Right now, the `<a>` tag in the string uses double quotes everywhere. You will n
 
 # --hints--
 
-You should remove all the `backslashes` (<code>\\</code>).
+You should remove all the backslashes (<code>\\</code>).
```

Comment:
I believe this now works fine on Crowdin now.
```suggestion
You should remove all the backslashes (`\`).
```

Reply:
https://freecodecamp.crowdin.com/translate/690087755819f2f67b26dbee3be0eb53/21404/en-esem?filter=basic&value=13#q=725612

I tagged you here - this appears to be an instance where that bug occurs again.

Could it be that we need to re-upload the file to run the parser again?

Reply:
I think when you send them an email, you ask them why it is broken again.